README.md

```diff
@@ -1,9 +1,17 @@
 # doc-code-samples
 
-This repository goal is to hold all code examples shown on the [Algolia.com documentation website](https://algolia.com/doc).
+This repository hols all code examples shown on the [Algolia.com documentation website](https://algolia.com/doc).
```

Comment:
all runnable? because we don't host the snippets

Reply:
Sorry I'm not sure I follow. Are there sandboxes that wouldn't run on CSB?

Reply:
I mean we don't host the non-sandboxes here

Reply:
Thanks for the clarification. I've updated the readme to be more precise in regards to what the repository contain.